### PR TITLE
Adding Oct updates blog

### PR DESCRIPTION
**Pull request by hpdang**

### CODE REVIEW

File: _posts/2018-11-06-october-updates.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+- [**Flair**](https://github.com/zalandoresearch/flair) Zalando recently released a new version of this open source Natural Language Processing framework, it now runs on both Linux and Mac, [click here to test!](https://github.com/zalandoresearch/flair). The project was initiated by [Zalando Research](https://research.zalando.com/). Flair gives users the ability to tag, classify and understand the meanings of email messages, customer responses, website comments, or any other scenario where users submit text feedback to be automatically classified or otherwise processed.
```
Comment (perploug) [inline _posts/2018-11-06-october-updates.md]: ..."Zalando Research" recently released a new version...

File: _posts/2018-11-06-october-updates.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Inside Zalando Open Source 
+- [**Zalando hosted a Hack Night at the Berlin Tech Hub to celebrate Hacktoberfest - the month of open source.**](https://github.com/zalando) The main event started off with a number of lightning talks by open source projects, followed by a hacking session, where Zalando engineers gathered as teams and worked on challenges under domains of machine learning, database and web plug-ins. Project maintainers were present to support participants completing their very first contribution and pull request.
```
Comment (perploug) [inline _posts/2018-11-06-october-updates.md]: The berlin office is not a tech hub :) it's just the Berlin office

**Review by perploug — COMMENTED** [review]
Looks good - added two suggestions for clarification

File: _posts/2018-11-06-october-updates.md (hunk 1 above)
Comment (baggerspion) [reply _posts/2018-11-06-october-updates.md]: - In Postgres Operator: "Our engineers do not need **to** manually configure..."

File: _posts/2018-11-06-october-updates.md (hunk 2 above)
Comment (baggerspion) [reply _posts/2018-11-06-october-updates.md]: - "The main event started with..." Use of "started off" is somewhat antiquated these days.

**Review by baggerspion — CHANGES_REQUESTED** [review]
Left some quick comments for you to resolve before this can be merged.

**Review by perploug — APPROVED** [review]

**Comment by perploug** [comment]
@therealpadams - your proposed changes have been added - so just need you to approve

**Review by baggerspion — APPROVED** [review]

**Pull request merged**